WalletWasabi.Gui/Controls/WalletExplorer/HistoryTabViewModel.cs

```diff
@@ -25,6 +25,7 @@ public class HistoryTabViewModel : WasabiDocumentTabViewModel, IWalletViewModel
 		private SortOrder _dateSortDirection;
 		private SortOrder _amountSortDirection;
 		private SortOrder _transactionSortDirection;
+		private SortOrder _labelSortDirection;
```

Comment:
Changes in this file is purely copypastas with adjusted variable name.

Reply:
ACK